pentagon/defaults.py

```diff
@@ -17,7 +17,7 @@ class AWSPentagonDefaults(object):
         'node_root_volume_size': 200,
         'v_log_level': 10,
         'network_cidr': '172.20.0.0/16',
-        'version': '1.8.4',
+        'kubernetes_version': '1.8.4',
```

Comment:
Would you also bump the version, it seems like a reasonable PR to update that.

Reply:
To the latest 1.8 or do we want to jump to 1.9.9?

Reply:
1.9.9 I think.

Reply:
👍 I'll bump it.